vdyp-sindex/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  
+  <artifactId>vdyp-sindex</artifactId>
+  
+  <name>Variable Density Yield Project - Sindex</name>
+  <url>http://maven.apache.org</url>
+
+  <parent>
+    <groupId>com.bc.gov.nrs.vdyp</groupId>
+    <artifactId>vdyp-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>

```

Comment:
Need to add a corresponding reference to the module in the parent pom in the root.